front-end-admin/src/components/pages/Home.vue

```diff
@@ -4,18 +4,17 @@
       <!-- Main jumbotron for a primary marketing message or call to action -->
       <div v-if="loggedIn" class="jumbotron">
         <div class="container">
+          <div>&nbsp;</div>
```

Comment:
what does this do?

Reply:
it adds a space. There might be a better way.